packages/fxa-auth-server/lib/routes/subscriptions.ts

```diff
@@ -184,7 +149,7 @@ class DirectStripeRoutes {
       paymentToken,
       displayName,
       idempotencyKey,
-    } = request.payload;
+    } = request.payload as any;
```

Comment:
Maybe a (future) opportunity here to share types with [payments-server's apiClient](https://github.com/mozilla/fxa/blob/main/packages/fxa-payments-server/src/lib/apiClient.ts#L125)?

Reply:
A future way to do this would be to re-type `validators.js` to TypeScript and swap `joi` for `typesafe-joi` which would allow us to get proper types from the existing joi schema. I used an approach like that in fxa-event-broker. Once that is done, then we could just indicate this should be typed as the validator for the payload, and woosh, type-safe access to what has been validated. Someday....